scene/2d/camera_2d.cpp

```diff
@@ -409,6 +409,35 @@ void Camera2D::force_update_scroll() {
 	_update_scroll();
 }
 
+void Camera2D::reset_smoothing() {
+
+	smoothed_camera_pos = camera_pos;
+	_update_scroll();
+}
+
+void Camera2D::align() {
+
+	Size2 screen_size = get_viewport_rect().size;
+	screen_size=get_viewport_rect().size;
```

Comment:
Ah okay, got it.
Yes, saw this, too, but it's in get_camera_transform(), too, so I thought it is kinda hack and it's needed to work. I didn't go deep into this, just modified the existing code.